lib/chainlock/chainlock.js

```diff
@@ -0,0 +1,278 @@
+var _ = require('lodash');
```

Comment:
Maybe we should put it to something like `lib/messages/chainlock.js`? I don't like `messages` but later we will add InstantSendLock and probably it makes sense to group them?

Reply:
Good idea, it also matches how we dir it in dashcore-p2p.